### PR TITLE
Excalibur dev

### PR DESCRIPTION
**Pull request by ajgdls**

### CODE REVIEW

File: odinApp/src/odinDataDriver.cpp [hunk 1]
```diff
@@ -120,56 +120,60 @@ void OdinDataDriver::configureOdinDataProcess(const char * ipAddress, int readyP
 
 int OdinDataDriver::createOdinDataParams()
 {
+  mFRConfiguration        = createODRESTParam(OdinFRConfig, REST_P_STRING,
+                                              SSDataFRConfig, "config_file");
   mProcesses              = createODRESTParam(OdinNumProcesses, REST_P_INT,
-                                              SSDataConfigHDFProcess, "number");
+                                              SSDataFWConfigHDFProcess, "number");
```
Comment (GDYendell) [inline odinApp/src/odinDataDriver.cpp]: Should these be `FP` instead of `FW`?

Comment (GDYendell) [reply odinApp/src/odinDataDriver.cpp]: We could take out the `Data` bit now too.

File: odinApp/Db/odinDataDriver.template [hunk 2]
```diff
@@ -228,21 +228,58 @@ record(bi, "$(P)$(R)FilePathExists_RBV")
...
+{
+    field(DTYP, "asynOctetRead")
+    field(INP,  "@asyn($(PORT),$(ADDR),$(TIMEOUT))FILE_NAME")
+    field(FTVL, "CHAR")
+    field(NELM, "256")
+    field(SCAN, "I/O Intr")
+}
+
+# File template
+record(waveform, "$(P)$(R)FileTemplate")
```
Comment (GDYendell) [inline odinApp/Db/odinDataDriver.template]: The FileTemplate PV can be removed now.

**Comment by ajgdls** [comment]
Yes to both, but @GDYendell can you make those changes or else you might need to wait until I'm back.
I realise now I used FW instead of FP for many of the enumerations, but I also think we need to look through all of those to make them consistent now that we have both FR and FP.

**Comment by GDYendell** [comment]
@ajgdls I will make those changes, check everything works and merge.

**Comment by GDYendell** [comment]
I think everything works as expected, except @ajgdls has some changes to the EDM screens that need committing. I have these locally for testing, so I am merging this.

**Pull request merged**